DABI.js

```diff
@@ -0,0 +1,321 @@
+            }
+        }
+    }
+
+    //Sometimes titles are missing
+    if (items["Untertitel"] && !items["Titel"]) {
+        newItem.title = items["Untertitel"].replace(/\*/g, '');
+        var short = newItem.title.replace(/^\W?(?:Die |Der |Das |\.{3}\s?)?/, '');
+        short = short.replace(/\W?(,|:|\?|!|\.|\"|\').*$/, '').split(' ').slice(0, 6).join(' ');
+        newItem.shortTitle = short.substring(0, 1).toUpperCase() + short.slice(1);
+    }
+
+    //Formatting and saving "Author" field
+    if (items["Autoren"]) {
+        var authors = items["Autoren"].split("; ");
+        for (var i = 0; i < authors.length; i++) {
+            newItem.creators.push(ZU.cleanAuthor(authors[i], "author", true));
+        }
+    }
+
+    //Formatting and saving "pages" field
+    if (items["Anfangsseite"]) {
+        newItem.pages = items["Anfangsseite"] + (items["Endseite"] ? "-" + items["Endseite"] : "");
+    }
+
+    //Saving the tags to Zotero
+    if (items["Schlagwörter"]) {
+        var tags = items["Schlagwörter"].split("; ");
+        for (var i = 0; i < tags.length; i++) {
+            newItem.tags.push(tags[i]);
+        }
+    }
+
+    //Making the publication title orthographic
+    if (items["Zeitschrift"]) {
+        newItem.publicationTitle = items["Zeitschrift"].replace(/ : /g, ": ");
+    }
+
+    //Associating and saving the well formatted data to Zotero
+    var fieldMap = {
+        "date": items["Jahr"],
+        "issue": items["Heft"],
+        "volume": items["Band"],
+        "abstractNote": items["Abstract"]
+    };
+
+    for (var key in fieldMap) {
+        if (fieldMap.hasOwnProperty(key)) {
+            newItem[key] = fieldMap[key];
+        }
+    };
+
+    newItem.libraryCatalog = "DABI";
+
+    //Scrape is COMPLETE!
+    newItem.complete();
+} /** BEGIN TEST CASES **/
+var testCases = [{
+        "type": "web",
+        "url": "http://dabi.ib.hu-berlin.de/cgi-bin/dabi/vollanzeige.pl?artikel_id=13028&modus=html",
+        "items": [{
+            "itemType": "journalArticle",
+            "title": "\"Mich interessierten kostengünstige Alternativen zu Citavi\": Über den Fortbildungsworkshop \"Literaturverwaltung im Fokus\" im Rahmen der AGMB-Tagung 2012",
+            "creators": [{
+                "firstName": "Matti",
+                "lastName": "Stöhr",
+                "creatorType": "author"
+            }],
+            "date": "2012",
+            "abstractNote": "Zum Programm der AGMB-Tagung 2012 in Aachen gehörte u.a. der zweistündige Fortbildungsworkshop \"Literaturverwaltung im Fokus - Softwaretypen, bibliothekarische Services und mehr\". Im Beitrag werden weniger die referierten Workshopinhalte beschrieben, als vielmehr die Perspektive der Teilnehmerinnen und Teilnehmer anhand einer eMail-basierten Umfrage vorgestellt. Die Kernfrage lautet hierbei: War der Workshop für sie gewinnbringend?",
+            "issue": "3",
+            "libraryCatalog": "DABI",
+            "pages": "0-0",
```

Comment:
Check for this case and make sure it doesn't happen. (1) if it's just `0`, we should leave pages blank. (2) If start and end is the same, it should just be start.

Reply:
doesn't happen anymore